storybook/stories/API/cartesian/CartesianAxis.stories.tsx

```diff
@@ -77,6 +77,78 @@ export const TickPositioning = {
   },
 };
 
+export const TickAlignment = {
```

Comment:
Nit: Given that this PR adds a new property, it would be great to also add a description to the Argument Table - as done in the Line.stories.tsx
Not required, but would be very well appreciated :)